apps/dotcms-ui/src/app/portlets/dot-starter/dot-starter.component.ts

```diff
@@ -62,7 +62,7 @@ export class DotStarterComponent implements OnInit {
     handleVisibility(hide: boolean): void {
         const layoutId = 'gettingstarted';
         hide
-            ? this.dotToolGroupService.hide(layoutId).pipe(take(1)).subscribe()
-            : this.dotToolGroupService.show(layoutId).pipe(take(1)).subscribe();
+            ? this.accountService.removeStarterPage().subscribe()
+            : this.accountService.addStarterPage().subscribe();
```

Comment:
What happens if one of this request fail?

Reply:
error in request and console

Reply:
Shouldn't the service that handle errors take care?

Reply:
@fmontes https://github.com/dotCMS/core-web/pull/1642/commits/253b22613a4f01d47ce306ab86430aca638bcf04